src/NServiceBus.Core.Tests/Serializers/XML/Issue_2796.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace NServiceBus.Core.Tests.Serializers.XML
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Serializers.XML.Test;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class Issue_2796
```

Comment:
Does the imply that the test should live in https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core.Tests/Serializers/XML/SerializerTests.cs ?

It's a huge class, but I would argue that that is where it belongs. We can think about splitting the class up, by I would rather do this based on logical capabilities, rather than issues.